app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -862,6 +868,9 @@ public void dismiss() {
     @SuppressWarnings("unused")
     private void handlePoorPerformance() {
         runOnUiThread(() -> {
+            if (!sPerformanceMonitorEnabled) {
```

Comment:
I think we could avoid all the static code in the activity just using here: 
`if (!mSettings.isPerformanceMonitorEnabled()) {`